Appraisals

```diff
@@ -0,0 +1,7 @@
+appraise "activesupport-6.0" do
+  gem "activesupport", "6.0"
+end
+
+appraise "activesupport-6.1" do
+  gem "activesupport", "6.1"
+end
```

Comment:
I like using `~>` and a `for` loop.
```suggestion
%w(6.0.4 6.1.4).each do |as_version|
  appraise "activesupport-#{as_version.split('.').first(2).join}" do
    gem "activesupport", " ~> #{as_version}"
  end
end
```

Stolen from `virtual_attributes`

Reply:
yeah, sorry, I actually started with that but felt it was needlessly complex.

Reply:
The `~>` is right though... it's doing 6.1.0 with what I have now.  Thanks.

Reply:
Ok, I updated to use `~>`.  I thought we could do the loop route if we add another appraisal.  It seems more readable to enumerate the two.